dataset1.py

```
@@ -0,0 +1,111 @@
+from torch.utils.data import Dataset
+from utils import get_data_path
+
+
+SYMBOLS = ['Customer_info', 'Discount_info', 'Marketing_info', 'Onlinesales_info', 'Tax_info.csv']  # !!! 수정 금지 !!!
+
+
+class InfoDataset(Dataset):
+    def __init__(self,
+                 is_training=True, 
+                 in_columns=['고객ID', '거래ID', '거래날짜', '제품ID', '제품카테고리', '수량', '평균금액', '배송료', '쿠폰상태',
+                            '성별', '고객지역', '가입기간', 'GST', '월', '쿠폰코드', '할인율', '거래금액'], 
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)